target/linux/mediatek/dts/mt7981b-h3c-magic-nx30-pro-nmbm.dts

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/dts-v1/;
+#include "mt7981b-h3c-magic-nx30-pro.dts"
+
+/ {
+	model = "H3C Magic NX30 Pro (NMBM layout)";
```

Comment:
` (NMBM layout)` does not apply to the model.

Reply:
I think it's ok to put the NMBM information there since https://github.com/openwrt/openwrt/pull/12770 did this also.

Reply:
I know that there are such examples in the OpenWrt code. However, the model field should not be used for any other comments.

Reply:
It is used to easily distinguish whether NMBM is enabled. Since the two modes are incompatible, any confusion will lead to bricking.

Reply:
> It is used to easily distinguish whether NMBM is enabled. Since the two modes are incompatible, any confusion will lead to bricking.

Sysupgrade checks and warn user if device is incompatible. If this is not enough, then we need to create a separate field. And not mold everything into a device model.